launchable/commands/split_subset.py

```diff
@@ -52,48 +82,74 @@ def split_subset(
         rest: str,
         base_path: str,
         same_bin_files: Optional[List[str]],
+        is_split_by_groups: bool,
+        is_split_by_groups_with_rest: bool,
+        split_by_groups_output_dir: click.Path,
+        is_output_exclusion_rules: bool,
 ):
     TestPathWriter.base_path = base_path
 
+    client = LaunchableClient(test_runner=context.invoked_subcommand, dry_run=context.obj.dry_run)
+
     class SplitSubset(TestPathWriter):
         def __init__(self, dry_run: bool = False):
+            self.split_by_groups_output_handler = self._default_split_by_groups_output_handler
+            self.split_by_groups_exclusion_output_handler = self._default_split_by_groups_exclusion_output_handler
+            self.is_split_by_groups_with_rest = is_split_by_groups_with_rest
+            self.split_by_groups_output_dir = split_by_groups_output_dir
             super(SplitSubset, self).__init__(dry_run=dry_run)
```

Comment:
[nits]
It's rather common to write `super().__init__()` first to override parent constructors. Even though it does not effect in this code, writing this phrase at last will make parent variables override child variables when the child class is meant override parant, which is nonsense.

Reply:
ok